### DISCUSSION TITLE
Debian 10 Buster: Vaultwarden 1.25 fails to start

### DISCUSSION THREAD

**Discussion by MatthiasKuehneEllerhold** [dani-garcia/vaultwarden#2550] [Q&A]
### Subject of the issue
Running Vaultwarden 1.25 under Debian 10 fails because `GLIBC_2.29' not found.

### Deployment environment
* vaultwarden version: 1.25.0
* Install method: Debian package from https://download.opensuse.org/repositories/home:/Masgalor:/Vaultwarden/Debian_10/
* other relevant information:
```
# strings /lib/x86_64-linux-gnu/libc.so.6 | grep GLIBC_
GLIBC_2.2.5
GLIBC_2.2.6
GLIBC_2.3
GLIBC_2.3.2
GLIBC_2.3.3
GLIBC_2.3.4
GLIBC_2.4
GLIBC_2.5
GLIBC_2.6
GLIBC_2.7
GLIBC_2.8
GLIBC_2.9
GLIBC_2.10
GLIBC_2.11
GLIBC_2.12
GLIBC_2.13
GLIBC_2.14
GLIBC_2.15
GLIBC_2.16
GLIBC_2.17
GLIBC_2.18
GLIBC_2.22
GLIBC_2.23
GLIBC_2.24
GLIBC_2.25
GLIBC_2.26
GLIBC_2.27
GLIBC_2.28
GLIBC_PRIVATE

# strings  /lib/x86_64-linux-gnu/libm.so.6 | grep GLIBC_
GLIBC_2.2.5
GLIBC_2.4
GLIBC_2.15
GLIBC_2.18
GLIBC_2.23
GLIBC_2.24
GLIBC_2.25
GLIBC_2.26
GLIBC_2.27
GLIBC_2.28
GLIBC_PRIVATE

# ldd --version
ldd (Debian GLIBC 2.28-10+deb10u1) 2.28
```

### Steps to reproduce
- Upgrade from 1.24.0 to 1.25.0 on a Debian 10 Buster machine
- Systemd cant start the vaultwarden service:
" /usr/bin/vaultwarden: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /usr/bin/vaultwarden)"

### Expected behaviour
Vaultwarden Service should start

### Actual behaviour
Vaultwarden Services refuses to start with the error message above.

**Comment by BlackDex** [comment]
I'm not sure how that binary is build. And it is not something done by the main project maintainers (as far as i know).

I suggest to see if you can contact the maintainer of that repo.

They need to use a different base for building the binary in this case.

**Reply by Masgalor** [reply]
I am the maintainer of that package.
It is indeed built with the wrong base system (Debian 11), I will fix this as soon as I have time for it.
